docs/sources/features/datasources/cloudmonitoring.md

```diff
@@ -81,11 +81,11 @@ Click on the links above and click the `Enable` button:
 
 If Grafana is running on a Google Compute Engine (GCE) virtual machine, it is possible for Grafana to automatically retrieve default credentials from the metadata server. This has the advantage of not needing to generate a private key file for the service account and also not having to upload the file to Grafana. However for this to work, there are a few preconditions that need to be met.
 
-1. First of all, you need to create a Service Account that can be used by the GCE virtual machine. See detailed instructions on how to do that [here](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#createanewserviceaccount).
-1. Make sure the GCE virtual machine instance is being run as the service account that you just created. See instructions [here](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#using).
+1. First of all, you need to create a Service Account that can be used by the GCE virtual machine. For more information about create a Service Account, refer to [`create new service account`](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#createanewserviceaccount).
+1. Make sure the GCE virtual machine instance is being run as the service account that you just created. For more information about use service account for instance, refer to [`using service account for instance`](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#using).
```

Comment:
Although "For more information about" is grammatically correct, we typically use "For more information on". Will you make the changes accordingly? Thanks a lot!

Reply:
Hi @achatterjee-grafana, thank you for your review..
Do you prefer to use "on" for all updates or just some parts that you have reviewed?

Reply:
The parts you made changes will. be fine. That will be a step toward consistency.

Reply:
sorry @achatterjee-grafana, may I confirm first with @oddlittlebird, because the instruction is using "about".
I've updated (as you have suggested) in my local, but wait for her confirmation first? :)

Reply:
Absolutely. As I said, you can use them interchangeably. But consistency is good :)

Reply:
I suggested "about" because I think it is less ambiguous in this context than "on."

Reply:
Thanks for the comment @oddlittlebird.  I do not agree about any ambiguity arising from the use of "on" in any of the statements. 

My point was we should be consistent and not change "about" and "on" interchangeably.